src/app/services/technical-record/technical-record.service.ts

```diff
@@ -257,4 +260,48 @@ export class TechnicalRecordService {
       }),
     );
   }
+
+  hasPsvGrossAxisChanged(changes: Partial<TechRecordGETPSV>): boolean {
+    return [
+      changes.techRecord_grossKerbWeight,
+      changes.techRecord_grossDesignWeight,
+      changes.techRecord_grossLadenWeight,
+      changes.techRecord_grossGbWeight,
+    ].some(Boolean);
+  }
+
+  hasHgvGrossAxisChanged(changes: Partial<TechRecordGETHGV>): boolean {
+    return [changes.techRecord_grossEecWeight, changes.techRecord_grossDesignWeight, changes.techRecord_grossGbWeight].some(Boolean);
+  }
+
+  hasTrlGrossAxisChanged(changes: Partial<TechRecordGETTRL>): boolean {
+    return [changes.techRecord_grossEecWeight, changes.techRecord_grossDesignWeight, changes.techRecord_grossGbWeight].some(Boolean);
+  }
+
+  hasHgvTrainAxisChanged(changes: Partial<TechRecordGETHGV>): boolean {
+    return [changes.techRecord_trainDesignWeight, changes.techRecord_trainGbWeight, changes.techRecord_trainEecWeight].some(Boolean);
+  }
+
+  hasPsvTrainAxisChanged(changes: Partial<TechRecordGETPSV>): boolean {
+    return [changes.techRecord_trainDesignWeight, changes.techRecord_maxTrainGbWeight].some(Boolean);
+  }
+
+  hasMaxTrainAxisChanged(changes: Partial<TechRecordGETHGV>): boolean {
+    return [
+      changes.techRecord_maxTrainDesignWeight,
+      changes.techRecord_maxTrainEecWeight,
+      changes.techRecord_maxTrainGbWeight,
+    ].some(Boolean);
+  }
+
+  hasAxisChanged(vehicleType: VehicleTypes, changes: Partial<TechRecordType<'get'>>) {
```

Comment:
Genuine question, why axis?

Reply:
This is probably on me, I now see this should probably be called 'haveAxlesChanged'. I imagine there are a fair few places where I've inadvertently used 'axis' instead of 'axles'.

Reply:
Nice one, thanks for fixing, nice work!